warehouse/csp.py

```diff
@@ -163,9 +163,10 @@ def includeme(config):
                 "font-src": [SELF, "fonts.gstatic.com"],
                 "form-action": [SELF, "https://checkout.stripe.com"],
                 "frame-ancestors": [NONE],
-                "frame-src": [NONE],
+                "frame-src": ["https://inspector.pypi.io"],
                 "img-src": [
                     SELF,
+                    "data:",  # Admin UI/Bootstrap 4 uses inline SVGs for icons
```

Comment:
Should we limit these to admin-only somehow?

Reply:
That's a good idea - I have a change that does that, similar to our CSP carveout for `/simple/`.